config-heroku.ini

```diff
@@ -1,12 +1,12 @@
 [General]
 # Instance name can be used to distinguish between different instances using backend api.getInstanceName()
-instanceName="heroku"
+instanceName="heroku-dafmdev"
 
 # set to true to allow using Trilium without authentication (makes sense for server build only, desktop build doesn't need password)
-noAuthentication=false
```

Comment:
I think `noAuthentication` in public network is not safe.
I would like to know your reasons for doing so